code/src/components/Header.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export const Header = () => {
```

Comment:
if you have a component which is only rendering something without doing any logic outside the return block you can use shorthand: 
``` 
export const Header = () => ( 
        <header className ='header'> 
            <h1 className="h1">New albums & singles</h1> 
        </header> 
    )
```